{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 1}, "tests": {"0": {"status": "ERROR", "message": "self = <Automation_framework.test.test_amazon.TestamazonWebsite object at 0x000001A418D458B0>\n\n    @pytest.fixture(autouse=True)\n    def setup(self):\n        self.aw = amazonwebsite(self.driver) # locator function class\n>       self.json_data = read_json_file()   #test data json file\n\ntest\\test_amazon.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nfilename = 'data/testcase_data.json'\n\n    def read_json_file(filename=\"data/testcase_data.json\"):\n>       with open(filename) as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'data/testcase_data.json'\n\nutilities\\utils.py:9: FileNotFoundError\n", "test_name": "test_login", "rerun": "0"}}, "suite_name": "test/test_amazon.py"}}}, "date": "April 30, 2024", "start_time": 1714467029.1232948, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "0", "skip": "0", "error": "1", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}